<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquityPriceUpdateConfDlg</class>
 <widget class="QDialog" name="EquityPriceUpdateConfDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Updating price configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Updating price policy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="m_updateAll">
        <property name="text">
         <string>Update all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_updateMissing">
        <property name="text">
         <string>Update &amp;missing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_updateDownloaded">
        <property name="text">
         <string>Update downloaded</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_updateSource">
        <property name="text">
         <string>Update from the same source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_ask">
        <property name="text">
         <string>Ask</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EquityPriceUpdateConfDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EquityPriceUpdateConfDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
